src/components/charts/CSVWrapper.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react"
+import { LineChart, LineChartProps } from "./LineChart"
+import { AreaChart, AreaChartProps } from "./AreaChart"
+import { BarChart, BarChartProps } from "./BarChart"
+import { csv as d3_csv } from "d3"
+
+interface LineChartCSVProps extends Omit<LineChartProps, "data"> {
+  /**
+   * Specifies the path to the CSV file.
+   */
+  data: string
```

Comment:
It seems odd to omit `data` and then just add back in the exact same parameter name. Perhaps `dataFile` or something?

Reply:
Makes sense 👍